apps/web/pages/api/user/avatar.ts

```diff
@@ -16,10 +17,18 @@ const querySchema = z
 
 async function getIdentityData(req: NextApiRequest) {
   const { username, teamname } = querySchema.parse(req.query);
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(req.headers.host ?? "");
```

Comment:
I think as a followup, we can cleanup this recomputation of orgDomain at all places. We are already computing it in middleware, so we can set it on req directly as a header maybe?